packages/web-components/src/components/logo-grid/logo-grid.ts

```diff
@@ -24,19 +22,11 @@ const { stablePrefix: ddsPrefix } = ddsSettings;
  */
 @customElement(`${ddsPrefix}-logo-grid`)
 class DDSLogoGrid extends StableSelectorMixin(DDSContentBlock) {
-  static get stableSelector() {
-    return `${ddsPrefix}--logo-grid`;
-  }
-
-  static get hideBorderClass() {
-    return `${prefix}--logo-grid__no-border`;
-  }
-
   @property({ attribute: 'hide-border', reflect: true, type: Boolean })
   hideBorder = false;
 
   protected updated() {
-    this.classList.toggle((this.constructor as typeof DDSLogoGrid).hideBorderClass, this.hideBorder);
+    this.toggleAttribute('hide-border', this.hideBorder);
   }
```

Comment:
`@property({ attribute: 'hide-border', reflect: true, type: Boolean })` takes care of this automatically.


```suggestion
```